product_docs/docs/pem/8/tuning_performance/performance_diagnostic.mdx

```diff
@@ -11,7 +11,9 @@ redirects:
 The Performance Diagnostic dashboard analyzes the database performance for Postgres instances by monitoring the wait events. To display the diagnostic graphs, PEM uses the data collected by the EDB Wait States module.
 
 !!! Note
-    For PostgreSQL databases, Performance Diagnostics is supported only for versions 10, 11, and 12 that are installed on the supported CentOS or RHEL platforms.
+    - For PostgreSQL databases, Performance Diagnostics is supported for versions 10 or later installed on the supported CentOS or RHEL platforms.
+
+    - For EDB Postgres Extended databases, Performance Diagnostics is supported for versions 11 or later on the supported CentOS or RHEL platforms.
 
 For more information on EDB Wait States, see [EDB Wait States](/epas/latest/epas_guide/13_performance_analysis_and_tuning/#edb-wait-states).
```

Comment:
while you're in here, can you lower case wait states both in the text and in the reference?

Reply:
edited